CourseDevelopment/Week8/Readme.MD

```diff
@@ -0,0 +1,194 @@
+# Course Development Week 8 The scientific process, beyond the PhD
+
+Course Development Week 8 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+
+Date: Feb/28/2023
+
+Name: Hendrik Snijder @hendelhendel
+
+Open tasks @SanliFaez @erikvansebille
++ Feedback on lecture-tutorial distribution of topics
++ Feedback on lecture format
++ Feedback on tutorial format
+
+## Goal: open design and open hardware; prototyping? 
+
+Dalle2 & @hendelhendel: A grant proposal and money for science, while a duck is doing research in her lab.
+![eendjefunding](https://user-images.githubusercontent.com/93695286/222175974-54ffc694-d1b4-4e7b-bdd7-7f9d8c94f2d8.png)
+
+## Keywords student familiarity check week 8
+
+|**New Concepts**|**Familiar Concepts**|
+|----------------|---------------|
+|Careers in science and beyond | Mental health in academia|
+|Applying for scientific positions | Literature and referencing|
+|Data stewardship|CV and applications|
```

Comment:
Not sure whether Data stewardship belongs in this week. Would that not fit better in Week 4 (or Week 3)?

Reply:
Hmm, I agree that, topic-wise, data management would fit better in Week 4. However, data stewardship and data governance is also an important part of grant writing. So there has to be something taught about data in Week8.